### DISCUSSION TITLE
Open hardware?

### DISCUSSION THREAD

**Discussion by IrregularShed** [DatanoiseTV/PicoDSP-Hardware#4] [General]
Hi

Is this open hardware? I was hoping to find a schematic but can't find one in any of the repositories. (I'm probably going to get one on Tindie but I'm always interested in this sort of thing!)

**Comment by DatanoiseTV** [comment]
Hi,
For now and probably for some more time the software is open-source but hardware schematics and board design are closed-source.

**Reply by IrregularShed** [reply]
Completely understandable and reasonable 🙂 I just wish your Tindie stock had lasted until my pay day!

**Reply by DatanoiseTV** [reply]
I am working on producing new ones in a week or two and restocking very soon.
Can check if I still have a board left from the previous run (green, no QWIIC connector, otherwise same specs).
